pkg/controllers/deprovisioning/drift.go

```diff
@@ -60,13 +59,8 @@ func (d *Drift) ShouldDeprovision(ctx context.Context, c *Candidate) bool {
 
 // ComputeCommand generates a deprovisioning command given deprovisionable machines
 func (d *Drift) ComputeCommand(ctx context.Context, nodes ...*Candidate) (Command, error) {
```

Comment:
```suggestion
func (d *Drift) ComputeCommand(ctx context.Context, candidates ...*Candidate) (Command, error) {
```

Maybe rename this to candidates at the top-level and just call it candidates everywhere?

Reply:
+1